emission/tests/storageTests/TestMongoGeoJSONQueries.py

```diff
@@ -112,11 +112,11 @@ def getTestNegPolygon(self):
             [-123,38],[-123,37]]
 
   def testGeoWithNegativeValues(self):
-    test = {'track_location': {'type':'Point', 'coordinates': [-122.5,37.5]} }
+    test = {'track_location': {'type':'Point', 'coordinates': [122.5,37.5]} }
```

Comment:
@swastis10 what is the motivation for this change?
I intentionally had it as a negative value to make sure that negative values worked. Note that all locations in the US have negative longitude. If this is breaking, we need to fix the code, not the test